robottelo/host_helpers/api_factory.py

```diff
@@ -592,17 +594,17 @@ def update_provisioning_template(self, name=None, old=None, new=None):
 
         :return bool: True/False
         """
-        temp = (
+        self.temp = (
             self._satellite.api.ProvisioningTemplate()
             .search(query={'per_page': '1000', 'search': f'name="{name}"'})[0]
             .read()
         )
-        if old in temp.template:
-            with templateupdate(temp):
-                temp.template = temp.template.replace(old, new, 1)
-                update = temp.update(['template'])
+        if old in self.temp.template:
```

Comment:
Now someone can understand it as a temporary template

Reply:
I will take this comment in different PR.